EIPS/eip-1418.md

```diff
@@ -0,0 +1,186 @@
+---
+eip: 1418
+title: Blockchain Storage Rent Payment
+author: William Entriken <@fulldecent>
```

Comment:
This seems to be an invalid formatting, usernames are in parenthesis.

Reply:
Thank you, resolved in 83672eafc9f287d24f36c479dec2810785f6a743